### ISSUE TITLE
Add support to ask for permissions in a custom slide

### ISSUE THREAD

**Issue by dsdebastiani** [TangoAgency/material-intro-screen#7]
If already have a way to do that. How to?

**Comment by bezmian** [comment]
Resolved in https://github.com/TangoAgency/material-intro-screen/pull/13/commits/c884c58e3f5bac68a85e9a110589b91d51b3434e.
In your custom slide, you need to add following lines to ask for permissions.
```java
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        neededPermissions = new String[]{Manifest.permission.CAMERA, Manifest.permission.ACCESS_FINE_LOCATION, Manifest.permission.ACCESS_COARSE_LOCATION};
    }
```

**Comment by radzio** [comment]
@bezmian please do not make neededPermissions protected / public. It is more clean to add method for this:

registerNeededPermission(String permission)
registerNeededPermisions(Collection<String> permissions)

**Comment by bezmian** [comment]
@radzio I have resolved it in the way you said.
To add permission in custom slide  you need to add following lines:
```java
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        registerNeededPermissions(new String[]{Manifest.permission.CAMERA, Manifest.permission.ACCESS_FINE_LOCATION, Manifest.permission.ACCESS_COARSE_LOCATION});
        registerPossiblePermissions(new String[]{Manifest.permission.CAMERA, Manifest.permission.ACCESS_FINE_LOCATION, Manifest.permission.ACCESS_COARSE_LOCATION});
    }
```

**Comment by nathan-osman** [comment]
`registerNeededPermissions()` is not a member of `SlideFragment`. Should I open a new issue for this?

The commit in question that fixed this (f5e87eb7640d2c0720dfff486c865e4663af0dd0) is missing from the repository.
